contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -2334,6 +2347,18 @@ static void bbf_ProcessUtility(PlannedStmt *pstmt,
 									 errmsg("The login name '%s' is too long. Maximum length is %d.",
 											stmt->role, (NAMEDATALEN - 1))));
 						}
+
+						/*
+						 * If the login name contains '\' and it is not a windows login then throw error.
+						 * For windows login, all cases are handled beforehand, so if the below condition
+						 * is hit that means it is password based authentication and login name contains
+						 * '\', which is not allowed
+						 */
+						if (strchr(stmt->role, '\\') != NULL)
```

Comment:
shouldn't this be `if (!from_windows && strchr(stmt->role, '\\') != NULL)`

Reply:
Done!